Sources/Solana/Actions/sendSOL/sendSOL.swift

```diff
@@ -56,11 +156,41 @@ extension Action {
         signer: Signer,
         onComplete: @escaping ((Result<TransactionID, Error>) -> Void)
     ) {
-        guard let to = PublicKey(string: destination) else {
-            onComplete(.failure(SolanaError.invalidPublicKey))
+        let instructionsResult = sendSolInstructions(from: fromPublicKey, to: destination, amount: amount, computeUnitLimit: computeUnitLimit, computeUnitPrice: computeUnitPrice)
+
+        let instructions: [TransactionInstruction]
+        switch instructionsResult {
+        case .success(let array):
+            instructions = array
+        case .failure(let error):
+            onComplete(.failure(error))
             return
         }
         
+        self.serializeAndSendWithFee(
+            instructions: instructions,
+            signers: [signer]
+        ) {
+            switch $0 {
+            case .success(let transaction):
+                onComplete(.success(transaction))
+            case .failure(let error):
+                onComplete(.failure(error))
+            }
```

Comment:
выглядит так, что можно 
```suggestion
            onComplete($0)
```